--- tests/test_backends_interactive.py.old	2017-05-30 09:29:32.000000000 +0100
+++ tests/test_backends_interactive.py	2017-05-31 12:38:31.000000000 +0100
@@ -21,10 +21,12 @@
 
 def _get_available_backends():
     return [
-        pytest.mark.skipif(sys.version_info < (3,)
-                           or importlib.util.find_spec(module_name) is None,
-                           reason="Could not import {!r}".format(module_name))(
-                               backend)
+        pytest.mark.skipif(
+            not os.environ.get("DISPLAY")
+            or sys.version_info < (3,)
+            or importlib.util.find_spec(module_name) is None,
+            reason="No $DISPLAY, or could not import {!r}".format(module_name))(
+                backend)
         for module_name, backend in [
                 ("PyQt5", "qt5agg"),
                 ("tkinter", "tkagg"),
@@ -41,8 +43,6 @@
 """
 
 
-@pytest.mark.skipif("DISPLAY" not in os.environ,
-                    reason="The DISPLAY environment variable is not set.")
 @pytest.mark.parametrize("backend", _get_available_backends())
 def test_backend(backend):
     environ = os.environ.copy()
