--- tests/test_backends_interactive.py.old	2017-05-30 09:29:32.000000000 +0100
+++ tests/test_backends_interactive.py	2017-05-30 09:31:19.000000000 +0100
@@ -46,6 +46,7 @@
 @pytest.mark.parametrize("backend", _get_available_backends())
 def test_backend(backend):
     environ = os.environ.copy()
+    assert environ.get('DISPLAY') is None
     environ["MPLBACKEND"] = backend
     proc = Popen([sys.executable, "-c", _test_script], env=environ)
     # Empirically, 1s is not enough on Travis.
