diff -rBNu matplotlib/testing/decorators.py matplotlib-patched/testing/decorators.py
--- matplotlib/testing/decorators.py	2016-12-29 15:27:32.000000000 -0800
+++ matplotlib-patched/testing/decorators.py	2016-12-29 15:36:10.000000000 -0800
@@ -358,7 +358,7 @@
                         file.close()
                 except ImportError:
                     # assume namespace package
-                    path = sys.modules[sub_mod].__path__
+                    path = list(sys.modules[sub_mod].__path__)
                     res = None, path, None
             return res
 
